flink-dist/src/main/flink-bin/bin/docker-entrypoint.sh

```diff
@@ -0,0 +1,176 @@
+#!/usr/bin/env bash
+
+###############################################################################
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+# limitations under the License.
+###############################################################################
+
+#######################################
+# This script is called by the Flink docker images when starting a process.
+# It contains docker-specific features, and hard-codes a few options.
+#
+# Globals:
+#   FLINK_HOME                          - (required) path to the Flink home directory
+#   ENABLE_BUILT_IN_PLUGINS             - semi-colon (;) separated list of plugins to enable, e.g., "flink-plugin1.jar;flink-plugin2.jar"
+#   FLINK_PROPERTIES                    - additional flink-conf.yaml entries as a multi-line string
+#   JOB_MANAGER_RPC_ADDRESS             - RPC address of the job manager
+#   TASK_MANAGER_NUMBER_OF_TASK_SLOTS   - number of slots for task executors
+#######################################
+
+COMMAND_STANDALONE="standalone-job"
+# Deprecated, should be remove in Flink release 1.13
+COMMAND_NATIVE_KUBERNETES="native-k8s"
+COMMAND_HISTORY_SERVER="history-server"
+
+args=("$@")
+echo "${args[@]}"
```

Comment:
We should not print any text in the pass through mode. Otherwise, the PR for docker-library/official-images will fail since CI will run the this test[1].

[1]. https://github.com/docker-library/official-images/blob/master/test/tests/override-cmd/run.sh

Reply:
my bad, this was for debugging.